### DISCUSSION TITLE
AutoGPT Template

### DISCUSSION THREAD

**Discussion by cycloner2020** [FlowiseAI/Flowise#112] [Q&A]
Hi there. I wanted to see if anyone has worked with the AutoGPT template yet.  I'm having issues getting to read or write files from Render. Do you have to setup a Disk on Render first or is there a way to write the files locally? Thanks in advance!

**Comment by goldzulu** [comment]
Haven't tried on render but it works fine on my local machine on Mac. I believe it's a matter of finding the right path try /var/lib/data. Or /usr/share/. Those might work. However it will not be persistent. You would best use Render disk and mount it on the path above so it will be persistent between sessions.hope that helps.

I'm trying to work on a S3 plug-in at the moment.

**Comment by Mobivs** [comment]
Yes, we need a good guide on that template!

**Reply by natelasko528** [reply]
Yes I am trying to learn how to create embeddings from websites, searches, etc to use with AutoGPT and Pinecone. Can anyone recommend a good guide. I filled in all of the info but my index is empty for my Pinecone setup but I have no idea how to make that happen. Can anyone please help?

**Comment by mwnu** [comment]
I deployed it on docker, but auto-gpt chatflow cannot read or write files locally. Mapping directories also doesn't work. The chat window only displays: "I have completed all my tasks."

**Comment by jerryselwyn** [comment]
it is possible to turn autogpt into custom tool. did anyone try it.
